co-simulation/lib/mosaic-geomath/src/main/java/org/eclipse/mosaic/lib/transform/Proj4Projection.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
```

Comment:
Should be Leidos copyright if this is a new addition for us.

Reply:
@adev4a. FYI, the copyright header in the Java code must exactly match the [template](https://github.com/usdot-fhwa-stol/cdasim/tree/develop/co-simulation/legal/templates). If the format is not strictly followed, MOSAIC will fail to load the Java code correctly

Reply:
Thank you for sharing! I added a new template for 2024 copyright and updated the text here